### PR TITLE
Mslq 82

### PR DESCRIPTION
**Pull request by harukins**
Ground Station page updated.

### CODE REVIEW

File: docs/source/software/ground_station.rst [hunk 1]
```diff
@@ -2,17 +2,39 @@
 Ground Station
 ===============
 
-This section is for all things involving ground based computers that are supporting the quad experiments. 
+This section is for all things involving ground-based computers that are supporting the quad experiments. Apart from your own personal computer, you also need to have Optitrack (our motion capture system running on relay.local and mocap.local in Stanford Flight Room) setup and streaming rigid body information. More information on how to setup the motion capture system can be found in the `Stanford Flight Room Documentation <https://stanfordflightroom.github.io/documentation>`_. In most cases, users only need to make changes on mocap.local, such as modifying and registering rigid bodies through the Motive Software.
```
Comment (exoticDFT) [inline docs/source/software/ground_station.rst]: In the style guide we are trying to follow an 80 character limit to lines when possible. So nothing major, but if we can break this sentence into separate lines that are 80 characters or less, that would be awesome! For the URLs that run longer than 80 characters, there isn't anything I think we can, or should, do for that. Just having them on a separate line should be enough. Same thing applies to some of the paragraphs below.

File: docs/source/software/ground_station.rst [hunk 2]
```diff
@@ -2,17 +2,39 @@
...
+   It makes the multitude of terminals that you will be needing a little
+   tidier.
 
-- Optitrack (our Motion Capture System)
-- Connecting to the Local Network
-- Using ROS
-- Our current Flightroom-specific computer setup 
-- and much, much, more
+ROS Melodic Installation and Setup
+----------------------------------
```
Comment (exoticDFT) [inline docs/source/software/ground_station.rst:26]: Super minor, but ReStructuredText prefers to have a blank line after any header/title. So just adding one here and after the other headers should be good.

File: docs/source/software/ground_station.rst [hunk 3]
```diff
@@ -2,17 +2,39 @@
...
+QGroundControl is used to configure the quads. To install and run QGroundControl on your personal computer, follow the Ubuntu Linux section of the `instructions page <https://docs.qgroundcontrol.com/master/en/getting_started/download_and_install.html#ubuntu>`_.
 
-These tools are useful if you want to monitor the status of the quad from your personal laptop, acquire near ground truth pose information of rigid bodies, or just synchronize your computer's clock with that of the quads. 
\ No newline at end of file
+
+Terminator Installation
+-----------------------
+This step is optional, although we highly recommend that you use Terminator or a similar tool to monitor multiple streams of ROS topics before and during flight. Terminator lets you arrange multiple terminals in grids and save layouts. See the `introduction page <https://gnometerminator.blogspot.com/p/introduction.html>`_ for more information and installation instructions.
+
+.. For information regarding:
```
Comment (exoticDFT) [inline docs/source/software/ground_station.rst]: I think it's safe to remove these comments, unless we want to have some extra subsection here that tells the user they can get more information related to the ground station by look at these topics and adding links they can go to. @k2shah @lowjunen what do you guys think?

**Review by exoticDFT — CHANGES_REQUESTED** [review]
Content looks great and reads well! Just a couple of style guide related comments and one bigger question about the comments you have at the end.

File: docs/source/software/ground_station.rst (hunk 1 above)
Comment (harukins) [reply docs/source/software/ground_station.rst]: Fixed in the new commit.

File: docs/source/software/ground_station.rst (hunk 2 above)
Comment (harukins) [reply docs/source/software/ground_station.rst:26]: Fixed in the new commit.

File: docs/source/software/ground_station.rst (hunk 3 above)
Comment (k2shah) [reply docs/source/software/ground_station.rst]: I think a "related links" section would be enough

**Review by k2shah — APPROVED** [review]

File: docs/source/software/ground_station.rst (hunk 3 above)
Comment (exoticDFT) [reply docs/source/software/ground_station.rst]: Looks great!

**Review by exoticDFT — APPROVED** [review]
Everything looks great!

File: docs/source/software/ground_station.rst (hunk 3 above)
Comment (harukins) [reply docs/source/software/ground_station.rst]: Will put the links in the external links section.

**Comment by harukins** [comment]
Ok, some comments in the ground station rst file have been moved to the external resources page, according to our previous discussion. I found the external resources page itself somewhat incomplete though. Could someone help edit it?

**Review by exoticDFT — CHANGES_REQUESTED** [review]
Changes look great and I think this is just about done. However, it seems that this branch was made before the recent correction to the external resources PR. So the conflicts that were made then are back again. We might need to merge the documentation branch into this before we actually finalize this PR.

File: docs/source/software/ground_station.rst (hunk 3 above)
Comment (exoticDFT) [reply docs/source/software/ground_station.rst]: Links moved to external resources look great. But I made a comment on the overall PR. I'm looking into this, but this conversation is good to resolve.

**Comment by k2shah** [comment]
this is being merged into documentation? do you want to rebase it before merge?

**Review by exoticDFT — APPROVED** [review]

**Pull request merged**